macos/FluentUI/Button/Button.swift

```diff
@@ -228,16 +233,15 @@ open class Button: NSButton {
 		}
 	}
 
-	private var restBackgroundColor: NSColor {
-		return fillColor
-	}
-
+	@objc public var restBackgroundColor: NSColor? = nil
```

Comment:
Would the default not be `fillColor` instead of `nil`? Also, can we not make this not an optional?